### ISSUE TITLE
Train issue

### ISSUE THREAD

**Issue by huxianer** [snakers4/mnasnet-pytorch#1]
@snakers4 could you tell me how many epochs do you use when it converge ,and the loss is not change at first.Thank you~

**Comment by snakers4** [comment]
Hm, strange, even with this network the loss should start decreasing on the first epoch...
